### PR TITLE
Fix javadoc references, links and spaces.

### PR DESCRIPTION
**Pull request by arturobernalg**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [ X] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by nvoxland** [comment]
**Initial/Pre-Review Thoughts**

Thanks for the fix. A quick scan looks right, but need someone to double check there isn't any unexpected changes to the javadocs

Questions I have:
- None

Potential risks:
- None

What could make the full review difficult:
- Nothing

**Review by MalloD12 — APPROVED** [review]
**Approved**

## Review and testing results:

PR changes looks good to me. Minor corrections (extra parenthesis removed from javadoc). Build and tests executed successfully (Functional tests failing at the moment, but that is not related with changes made here).

### Things to be aware of:
- None

### Things to worry about:
- None

**Review by filipelautert — APPROVED** [review]

**Comment by XDelphiGrl** [comment]
Functional tests running here: [https://github.com/liquibase/liquibase-pro-tests/actions/runs/4928321834](https://github.com/liquibase/liquibase-pro-tests/actions/runs/4928321834)
Test Harness running here: [https://github.com/liquibase/liquibase-test-harness/actions/runs/4928326159]

File: liquibase-standard/src/main/java/liquibase/ExtensibleObject.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 /**
  * This interface defines how objects can be extended with additional attributes at runtime without subclassing and exposes the ability to query attributes without resorting to reflection.
- *
+ * <p>
```
Comment (XDelphiGrl) [inline liquibase-standard/src/main/java/liquibase/ExtensibleObject.java:8]: For those who were unaware, as I was prior to this review, the parser infers the closing `</p>`. :)

**Review by XDelphiGrl — APPROVED** [review]
Do you like cleaner JavaDoc output, with correctly formatted URLs and clean spacing between paragraphs? You're in luck! This PR provides just that, with the addition of fixing two places in the documentation where the incorrect variable name was referenced in the JavaDoc annotations.
* Functional and test harness executions passing.
* No additional testing required.

APPROVED

**Pull request merged**